third_party/move/move-compiler/src/parser/keywords.rs

```diff
@@ -40,6 +40,7 @@ pub const CONTEXTUAL_KEYWORDS: &[&str] = &[
     "assume",
     "axiom",
     "choose",
+    "curry",
```

Comment:
Are there any tests on how these keywords are used?

Reply:
Note that these keywords are only used in docgen to highlight in documentation comments.  Currently they are not used at all, but I thought I should put a stake in the ground.